### PR TITLE
Migrate to claranet/php parent image

### PR DESCRIPTION
**Pull request by tonyfahrion**
Spryker base is split in two parts now. The generic claranet/php
image, which provides a generic build concept and preparations for
PHP based application stacks and all spryker specific steps / configs
are now merged into the claranet/spryker-demoshop docker image.

It is easier to maintain a generic PHP parent image and to see all
spryker related steps in one place within claranet/spryker-demoshop.
So we decided to switch the parent image.

In addition this merges a lot of lessons learned from running spryker
shops in production, unify configuration naming, enhance performance
settings and make the init process more stable.

Please have a look at the CHANGELOG-claranet.md for more details about
this update.

### CODE REVIEW

**Comment by tonyfahrion** [comment]
README and CHANGELOG-claranet are still in development; but the shop itself is tested and should be ready

**Comment by tonyfahrion** [comment]
okay, ready for review; mergeable from my point of view

File: README.md [hunk 1]
```diff
@@ -79,7 +82,7 @@ run stacks in parallel and prevent port collisions we need to align port
 allocation.
 
 Therefore the following scheme has been implemented: The port number is encoded
-like this: **ECCDD**
```
Comment (fuhbar) [inline README.md]: @tonyfahrion why removing the generic letter encoded form here? You've written `2CC00` instead. Why?

Comment (tonyfahrion) [reply README.md]: we aren't able to differentiate between stores `DD` via port; except if we change the config_local.php for local-dev only. But as far as I can tell, the user should be able to switch stores via the webui or by adding a different hostname, if required.

**Pull request merged**